components/41-organisms/timeline/_timeline.scss

```diff
@@ -229,7 +229,7 @@ dl.timeline {
       width: 100%;
       max-width: none;
       margin: 0;
-      padding: 0 calc(#{$gutter-width/2}) 0 calc(#{$timeline-m-left-padding} + #{$timeline-m-right-padding});
+      padding: 0 calc(#{$gutter-width*.5}) 0 calc(#{$timeline-m-left-padding} + #{$timeline-m-right-padding});
```

Comment:
Add space between * and .5. Makes it clearer that it is a calculation instead of a wildcard.